jsonrpc/server.go

```diff
@@ -397,7 +397,11 @@ func isNil(i any) bool {
 }
 
 func (s *Server) handleRequest(ctx context.Context, req *Request) (*response, error) {
+	reqJSON, _ := json.Marshal(req) // Serialise the request early for use in logging
```

Comment:
If we marshal here, we don't need to again at line 444

Reply:
But one is for request and the other for response, can't that cause soem errors?